### PR TITLE
release(map): Add 20.04.3 release note

### PR DESCRIPTION
**Pull request by jdelpierre**

### CODE REVIEW

**Comment by tuntoja** [comment]
Missing the other fix for retrieving secured image centreon (by jiliang)

File: en/releases/centreon-commercial-extensions.md [hunk 1]
```diff
@@ -18,6 +18,10 @@ If you have feature requests or want to report a bug, please contact support.
 
 ## Centreon MAP release notes
 
+### 20.04.3
+
+* Filter on the full name of the maps and geoviews on the homepage
+
```
Comment (tuntoja) [inline en/releases/centreon-commercial-extensions.md:24]: Fix on media access from centreon map server

File: fr/releases/centreon-commercial-extensions.md [hunk 2]
```diff
@@ -18,6 +18,10 @@ commerciales, veuillez contacter le support.
 
 ## Centreon MAP release notes
 
+### 20.04.3
+
+* Filtre sur le nom complet des cartes et geoviews sur la page d'accueil
+
```
Comment (tuntoja) [inline fr/releases/centreon-commercial-extensions.md:24]: Correction de l'accès aux media depuis centreon map server

**Review by tuntoja — COMMENTED** [review]
missing entry fo the other 20.04.3 resolved ticket

File: en/releases/centreon-commercial-extensions.md (hunk 1 above)
Comment (jdelpierre) [reply en/releases/centreon-commercial-extensions.md:25]: Ok I push it

**Review by thiuyendang — APPROVED** [review]

**Pull request merged**